mmdeploy/pytorch/functions/any.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(func_name='torch.Tensor.any')
+@FUNCTION_REWRITER.register_rewriter(func_name='torch.any')
+def any__default(input, *args, **kwargs) -> torch.Tensor:
```

Comment:
add dim and keepdim as parameters with default value here. so you don't have to check these params below.